modules/test/playwright/tests/users-admin-web/gdpr.spec.ts

```diff
@@ -299,3 +306,130 @@ testAdmin.describe('LPD-27068 Refactor of GDPR#CanAnonymizeAllEntries', () => {
 		}
 	);
 });
+
+testAdmin(
+	'LPD-31206 - Can delete a single staged and live blogs entry',
```

Comment:
In a next PR can you update this to remove the - (I think we are not using it)

Reply:
I'll change it next time, thanks!